content/documentation/validations/index.adoc

```diff
@@ -687,10 +687,156 @@ include::/data/files/validation_examples/601.yaml[]
+
+This validation refers to the scenario where there aren't workloads matching the labels specified into the sidecar's `workloadSelector` field. Therefore, all the settings defined within the sidecar resource won't be applied to any workload in the mesh.
+
+==== Resolution
+
+The validation will be gone when there is at least one workload matching the labels set in the `workloadSelector` field. The workload must be in the same namespace of the sidecar resource.
+
+==== Severity
+
+icon:exclamation-triangle[] Warning
+
+==== Example
+
+[source, yaml]
+----
+include::/data/files/validation_examples/901.yaml[]
+----
+
+==== See Also
+
+https://github.com/kiali/kiali/tree/master/business/checkers/sidecars/workload_selector_checker.go[Validator source code, window="_blank"] +
+https://istio.io/docs/reference/config/networking/sidecar/[Sidecar documentation, window="_blank"] +
+
+'''
+
+=== KIA1002 - More than one selector-less Sidecar in the same namespace
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. This resource is aimed to target either specific workloads, all workloads in a namespace or all the sidecars of the service mesh. Each Sidecar resource living in a namespace (except the one living in the control plane namespace) effects only to the workloads in the same namespace.
+
+Selector-less sidecars are those sidecars that doesn't have the `workloadSelector` field and thus they are pointing to all workloads running in the same namespace.
+Therefore, this validation refers to the scenario where there are more than one sidecar targetting to all workloads of the namespace. According to Istio documentation the behavior of the mesh is undefined when this scenario is presented.
+
+==== Resolution
+
+Make sure that all that there is only one selector-less sidecar in the namespace. Three possible solutions are suggested: merging selector-less sidecars into one, adding selectors to existing selector-less sidecars or removing selector-less sidecars.
+
+==== Severity
+
+icon:times-circle[] Error
+
+==== Example
+
+[source, yaml]
+----
+include::/data/files/validation_examples/902.yaml[]
+----
+
+==== See Also
+
+https://github.com/kiali/kiali/tree/master/business/checkers/sidecars/multi_match_checker.go[Validator source code, window="_blank"] +
+https://istio.io/docs/reference/config/networking/sidecar/[Sidecar documentation, window="_blank"] +
+
+'''
+
+=== KIA1003 - Invalid host format. 'namespace/dnsName' format expected
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. IstioEgressListener specifies the properties of an outbound traffic listener on the sidecar proxy attached to a workload instance.
+
+In the host field, there is the list of hosts exposed to the workload. Each host in the list must have the `namespace/dnsName` format.
```

Comment:
"In the host field, it is the list of host exposed" ?
Not sure if this sounds better, anyway not a blocker.

Reply:
> The hosts list is the list of hosts that will be exposed to the workload.

Better?